builds/release/refresh-metrics-collector-images.yaml

```diff
@@ -326,28 +329,24 @@ stages:
     
     - pwsh: |
         $testDir = '$(Build.SourcesDirectory)/test/Microsoft.Azure.Devices.Edge.Test'
-        dotnet build -c $(Build.Configuration) $testDir
+        dotnet build -c Release $testDir
```

Comment:
Please feel free to remove the variable `$(Build.Configuration)` from the pipeline.

Reply:
Yep, I have a list with all the changes I need to make to all the pipeline variables when this gets merged. ✅